src/components/stepFour/index.js

```diff
@@ -513,7 +512,7 @@ const { PUBLISH } = NAVIGATION_STEPS
     const tokenAddr = token.addr
     const crowdsaleAddr = crowdsale.addr
 
-    setLastCrowdsaleRecursive(pricingStrategyABI, pricingStrategy.addr, crowdsaleAddr.slice(-1)[0])
+    setTierRecursive(pricingStrategyABI, pricingStrategy.addr, crowdsaleAddr)
```

Comment:
I'm curious about this change, why were we passing the last tier only and now we are passing all the tiers?

Reply:
We were storing `lastCrowdsale` property for `updateRate` method to check whether crowdsale is finalized or not until the audit. I moved updateRate method to crowdsale contract to have only one endpoint to change crowdsale data. Since that, we don't need to store `lastCrowdsale` property, but we need to join crowdsale with corresponding pricing strategy. This is why we need to store tier address in pricing strategy contract.